src/signatures/ternary/mod.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(deprecated)]
+
+#[cfg(feature = "wots_deprecated_do_not_use")]
+#[cfg_attr(docsrs, doc(cfg(feature = "wots_deprecated_do_not_use")))]
+#[cfg_attr(not(test), deprecated)]
+pub mod wots;
+
+use crate::hashes::sponge::HASH_LENGTH;
+
+use bee_ternary::{T1B1Buf, TritBuf, Trits, T1B1};
+
+use zeroize::Zeroize;
+
+/// Length of a message fragment.
+pub const MESSAGE_FRAGMENT_LENGTH: usize = 27;
```

Comment:
This is actually the `MESSAGE_FRAGMENT_LENGTH` in trytes. Since most of the parameters (including `HASH_LENGTH` and `SIGNATURE_FRAGMENT_LENGTH`) are given in trits.
One could call that `MESSAGE_FRAGMENT_LENGTH_TRYTES` or clarify it in the comment.